src/main/kotlin/mate/academy/model/figure/Figure.kt

```diff
@@ -0,0 +1,7 @@
+package mate.academy.model.figure
+
+import mate.academy.model.enums.Color
+import mate.academy.service.interfaces.AreaCalculator
+import mate.academy.service.interfaces.Drawable
+
+abstract class Figure(val color: Color) : AreaCalculator, Drawable
```

Comment:
Using AreaCalculator, Drawable interfaces leads to code duplications in the child classes. Please consider using of abstract functions inside Figure class